integration-tests/src/lib.rs

```diff
@@ -21,4 +21,19 @@ mod staking;
 mod transact_call_filter;
 mod vesting;
 mod xcm_defer;
+mod xcm_rate_limiter;
+
+#[macro_export]
+macro_rules! assert_balance {
```

Comment:
Why do we want to have these macros here? I'm just curious. I wouldn't expect any code in this file.